notices.json

```diff
@@ -37,21 +37,21 @@
     }
   },
   {
-    "id": "server_upgrade_v5.31",
+    "id": "server_upgrade_v5.32",
     "conditions": {
       "audience": "sysadmin",
       "clientType": "all",
-      "serverVersion": ["<5.31"],
+      "serverVersion": ["<5.32"],
       "instanceType": "onprem",
-      "displayDate": ">= 2021-01-21T00:00:00Z"
+      "displayDate": ">= 2021-02-21T00:00:00Z"
     },
     "localizedMessages": {
       "en": {
         "title": "New server version available",
-        "description": "The new [Extended Support Release](https://docs.mattermost.com/administration/extended-support-release.html) v5.31 is now available offering bug fixes for increased stability and improvements in Incident Management and plugins. [Upgrading](https://docs.mattermost.com/administration/upgrade.html) only takes a few minutes.",
+        "description": "The new The new feature release offers a streamlined license renewal experience, a more specific incident status and task list, and custom, collapsible channel categories. [Upgrading](https://docs.mattermost.com/administration/upgrade.html) only takes a few minutes.",
```

Comment:
The new feature release offers a streamlined license renewal experience, updated incident status and task list, and custom, collapsible channel categories.